packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -525,34 +560,28 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
             <span className={styles.columnLabel}>{t('image', 'Image')}</span>
           </Column>
           <Column sm={3}>
-            <FormGroup legendText="">
+            <FormGroup>
               <p className={styles.imgUploadHelperText}>
-                {t('imageUploadHelperText', "Upload an image or use this device's camera to capture an image")}
+                {t('imageUploadHelperText', "Upload images or use this device's camera to capture images")}
               </p>
-              {currentImage?.base64Content ? (
-                <Button
-                  className={styles.uploadButton}
-                  kind={isTablet ? 'ghost' : 'tertiary'}
-                  onClick={() => showImageCaptureModal()}
-                  renderIcon={(props) => <Edit size={16} {...props} />}
-                >
-                  {t('changeImage', 'Change image')}
-                </Button>
-              ) : (
-                <Button
-                  className={styles.uploadButton}
-                  kind={isTablet ? 'ghost' : 'tertiary'}
-                  onClick={() => showImageCaptureModal()}
-                  renderIcon={(props) => <Add size={16} {...props} />}
-                >
-                  {t('addImage', 'Add image')}
-                </Button>
-              )}
-              {currentImage?.base64Content && currentImage?.fileType == 'image' ? (
-                <div className={styles.imgThumbnailContainer}>
-                  <img src={currentImage.base64Content} className={styles.imgThumbnail} />
-                </div>
-              ) : null}
+              <Button
+                className={styles.uploadButton}
+                kind="ghost"
+                onClick={showImageCaptureModal}
+                renderIcon={(props) => <Add size={16} {...props} />}
+              >
+                {t('addImage', 'Add image')}
+              </Button>
+              <div className={styles.imgThumbnailContainer}>
+                {uploadedImages.map((image, index) => (
+                  <div key={index} className={styles.imgThumbnailContainer}>
+                    <img src={image.base64Content} className={styles.imgThumbnail} alt={`Image ${index}`} />
+                    <Button kind="ghost" onClick={() => handleRemoveImage(index)}>
```

Comment:
I think visually, we need to be sure that there's some separation between the image and the button, i.e., the button should probably be below the image.